src/views/project/publishPartition/PublishPartitionView.vue

```diff
@@ -0,0 +1,90 @@
+<script setup>
+import axios from 'axios'
+import router from '@/router'
+import { ref, onMounted } from 'vue'
+import { useRoute } from 'vue-router'
+
+const route = useRoute()
+const projectId = route.params.id
+let partitions = ref([])
+
+onMounted(() => {
+  getPartitions()
+})
+
+async function getPartitions() {
+  const url = `${
+    import.meta.env.VITE_API_URL
+  }/projects/${projectId}/publish/partition`
+  const response = await axios.get(url)
+
+  partitions.value = response.data
```

Comment:
For very slow network connections, the page will first flash with the text "Currently, there are no partitions associated with this project.". Can you add a LoadingIndicator for this page too.